pubspec.yaml

```diff
@@ -14,7 +14,7 @@ description: Mobile Nebula Client
 version: 0.1.0+54
 
 environment:
-  sdk: ^3.5.1
+  sdk: ^3.7.0
```

Comment:
If you're planning to gitignore this, it should probably be only formatting changes, not version updates or anything functional.

Reply:
rebase on main fixed this diff, github is weird sometimes...